benchmarks/BUILD

```diff
@@ -0,0 +1,20 @@
+load("//bazel:rules.bzl", "cris_cc_test")
+
+cris_cc_test(
+    name = "benchmark",
+    srcs = [
+            "benchmark.cc"
+        ] + glob([
+            "*_benchmark.hpp",
+        ]),
+    deps = [
+        "//:sched",
+        "//:timer",
+        "//:utils",
+    ],
+    linkopts = ["-lbenchmark"],
+    env = {
+        # Mute INFO logs.
+        "GLOG_minloglevel": "1",
```

Comment:
Can we do this in cpp? Having non-build configs in build system is not a good idea.